bin/fingerprint

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+#
+# Print out the sha256 fingerprint for a x509 certificate.
+# Uses both the Fingerprinter Ruby library used by the app,
+# and the equivalent openssl command, just for comparison.
+# Usage:
+#  bin/fingerprint path/to/file.cert
+#
+
+require_relative '../lib/fingerprinter'
+
+ARGV.each do |cert|
+  cert_pem = File.read(cert)
+  fprint = Fingerprinter.fingerprint_cert(cert_pem)
+  puts "#{cert} => #{fprint.upcase.split(/(..)/).reject {|c| c.length == 0 }.join(':')}"
+  system("openssl x509 -sha256 -noout -fingerprint -in #{cert}")
+end
```

Comment:
You might argue that the script itself is overkill, and we could just include the one-liner openssl in the README. I kept forgetting where it was, and wanted to verify that the Fingerprinter code was doing the same thing. I'm happy to just keep this script locally if it is not useful for others.